features/miniAlert.js

```diff
@@ -0,0 +1,25 @@
+//Thanks Vals
+function inDungeons() {
+    if(!(Scoreboard.getTitle().trim() != "" && Scoreboard.getLines().length > 0)) {
+        return;
+    }
+    for(let i = 0; i < Scoreboard.getLines().length; i++) {
+        let line = Scoreboard.getLines()[i]
+        if(line.getName().includes('§cThe Cata') && line.getName().includes('mbs §8('))
+            return true
+    }
+    return false
+}
+register("renderWorld", () => {
+    if(settings.getSetting("Shit", "MoreShit") && inDungeons()) {

```

Comment:
the inDungeons should probably not be executed every frame
it should be executed every tick instead
I'm pretty sure that it's better for the performance

Reply:
also properly setup the settings

Reply:
oh also same applies to all the entity checking.

Reply:
Didn't Ask.